admin-manual/installation-setup/customization/scaling-archivematica.rst

```diff
@@ -0,0 +1,397 @@
+file. This might be advantageous where certain commands tend to be run on 
+certain kinds of objects, allowing you to route particular types of work to 
+specific MCPClients or machines. 
+ 
+Distribute other components to another machine 
+++++++++++++++++++++++++++++++++++++++++++++++
+
+It is possible to deploy the Elasticsearch, Gearman and MySQL components on 
+other machines.  
+For help, ask on the `Archivematica user forum`_ for more details.
+
+Optimizing settings across machines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Optimize batch size for large files
++++++++++++++++++++++++++++++++++++
+
+This technique will work just as well on multiple machines as it does on one 
+machine, as described above in :ref:`Scaling up: optimising on one machine <scaling-up>`. 
+
+Adjusting timeouts
+++++++++++++++++++
+
+Timeout settings are an important tool to mitigate failure scenarios created 
+when one component can't connect to another. The challenge is to set timeouts 
+so that they are long enough to allow particular processes to complete, but 
+not so long that system resources are left idle (or user's time is wasted), 
+waiting for a response from another component that has failed or can't be 
+reached due to network connectivity issues.  
+
+The standard timeout parameters for each component are used for long-running 
+(generally asynchronous) processes. "Quick" timeout values are for processes 
+that are synchronous and short (for instance, when an API is called to return 
+information to the UI for a waiting user). 
+
+The default value for the "Quick" timeouts is optimal for components that are 
+located on the same machine, and will be adequate in many cases for components 
+that are distributed to machines that are co-located. 
+
+There may be times when timeout values should be increased for distributed 
+components that are not co-located, or are slower due to the nature of the 
+communication protocol used. 
+
+See `Dashboard`_ Configuration, `MCPClient`_ Configuration, and `MCPServer`_ 
+Configuration for a list of all timeout parameters, their default settings and 
+instructions for modifying them.  
+
+Optimizing storage locations
+++++++++++++++++++++++++++++
+
+The Storage Service Administrator manual describes the different types of 
+:ref:`storage locations <storageService:locations>` that Archivematica uses. 
+
+In many cases it may be necessary to use different machines for different types
+of storage locations. In general, we recommend having the most frequently used 
+locations (e.g. the "currently processing" location) on a local machine. 
+Locations that are used less frequently, such as AIP or DIP storage, will have 
+less impact on performance when distributed to remote storage locations.  
+
+.. _config-strategies:
+
+Process configuration strategies 
+--------------------------------
+
+Optimising what and how preservation actions get executed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Travis is complaining about this line not being long enough:

```Warning, treated as error:
/home/travis/build/artefactual/archivematica-docs/admin-manual/installation-setup/customization/scaling-archivematica.rst:247

Title underline too short.

Optimising what and how preservation actions get executed
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^```

Reply:
oops, because I replaced & with 'and' and didn't update the underline. Fixing now.